scss/_portfolio.scss

```diff
@@ -1,40 +1,40 @@
-.portfolio {
-  padding: 8% 2% 5% 2%;
-  background-color: #f7f7f7;
-}
-
-.portfolio__heading {
-  @include section-heading;
-}
-
-.portfolio__main-paragraph {
-  @include paragraph-under-main-heading;
-}
-
-.portfolio__projects {
-  padding-top: 5.5%;
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: space-around;
-  text-align: center;
-  @include section-width;
-}
-
-.portfolio__col {
-  max-width: 291px;
-  margin: 0 2% 4.6% 2%;
-  background-color: #fff;
-}
-
-.portfolio__col-heading {
-  padding-top: 7%;
-  font: 1.125rem $font-main;
-  font-weight: 700;
-  color: #333333;
-}
-
-.portfolio__col-paragraph {
-  padding-bottom: 10%;
-  font: italic 0.875rem $font-secondary;
-  color: #737373;
+.portfolio {
+  padding: 8% 2% 5% 2%;
+  background-color: #f7f7f7;
+}
+
+.portfolio__heading {
+  @include section-heading;
+}
+
+.portfolio__main-paragraph {
+  @include paragraph-under-main-heading;
+}
+
+.portfolio__projects {
+  padding-top: 5.5%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  text-align: center;
+  @include section-width;
+}
+
+.portfolio__col {
+  max-width: 291px;
+  margin: 0 2% 4.6% 2%;
+  background-color: #fff;
+}
+
+.portfolio__col-heading {
+  padding-top: 7%;
+  font: 1.125rem $font-main;
+  font-weight: 700;
+  color: #333333;

```

Comment:
Somewhere before you wrote #fff, now #333333 but could #333. Both are correct but consider unifying.